src/DataLayer/KeyMotorData/I_KeyMotorData.h

```diff
@@ -19,8 +19,8 @@ class I_KeyMotorData : public QObject
 signals:
     void motorZeroReceived(KeyMotor motorZero);
```

Comment:
Let's keep consistency and have all signals be `void signal(DataType)`, without a variable name.